main.js

```diff
@@ -0,0 +1,122 @@
+// declare arrays
+var sides = ["Miso Glazed Carrots",
+"Coleslaw",
+"Garden Salad",
+"Crispy Potatoes",
+"Sweet Potato Tots",
+"Coconut Rice",
+"Caeser Salad",
+"Shrimp Summer Rolls",
+"Garlic Butter Mushrooms",
+"Hush Puppies"]
+
+var mains = [
+"Spaghetti and Meatballs",
+"Pineapple Chicken",
+"Shakshuka",
+"Thai Yellow Curry",
+"Bibimbap",
+"Chicken Parmesean",
+"Butternut Squash Soup",
+"BBQ Chicken Burgers",
+"Ramen",
+"Empanadas",
+"Chicken Fried Rice",
+"Sheet Pan Fajitas",
+"Margarita Pizza"
+]
+
+var desserts = [
+"Apple Pie",
+"Lemon Meringue Pie",
+"Black Forest Cake",
+"Banana Bread",
+"Peach Cobbler",
+"Cheesecake",
+"Funfetti Cake",
+"Baklava",
+"Flan",
+"Macarons",
+"Macaroons",
+"Chocolate Cupcakes",
+"Pavlova",
+"Pumpkin Pie",
+"Key Lime Pie",
+"Tart Tatin",
+"Croissants",
+"Eclairs"
+]
+//selectors
+var cookBtn = document.querySelector(".lets-cook")
+var radioBtns = document.getElementsByName('meal')
+var whatToMakeSection = document.querySelector('.what-to-make')
+var whatToMakeOutput = document.querySelector(".random-items")
+var cookpot = document.querySelector(".cookpot-image")
+
+//Event listen'n
+cookBtn.addEventListener('click', letsCook)
+
+
+
+
+
+//lets write some functions
+function letsCook() {
```

Comment:
I hate this. And yet it functions. Can I make it function without hating it? Tune in to Saturday's pull requests to find out.